src/Illuminate/Contracts/Cache/Store.php

```diff
@@ -83,6 +83,11 @@ public function forget($key);
      */
     public function flush();
 
+    /**
+     * Set the expiration of a cached item.
+     */
+    public function touch(string $key, int $ttl): bool;
```

Comment:
Adding a method to a public interface is a major breaking change.

Reply:
this is targetting `master` now, should be fine.